raspberry_pi/update.sh

```diff
@@ -42,7 +42,7 @@ cd ${src_home}
 user=$(whoami)
 group=$(id -gn)
 if [ -d ./logs ]; then
-    sudo chown ${user}:${group} ./logs/*
+    sudo chown ${user}:${group} ./logs/* || true
```

Comment:
This change is unrelated to the config changes, but was small enough that I didn't think it hurt mixing it in.